libs/api/src/lib/memberContext.ts

```diff
@@ -694,14 +700,66 @@ export class MemberContext implements MemberContextInterface {
       throw new InternalError()
     }
 
-    const invoice = await this.renewSubscriptionForUser({subscription})
+    if (startsAt < now || paidUntil) {
+      const endsAt = paidUntil ?? getNextDateForPeriodicity(startsAt, paymentPeriodicity)
 
-    // Send subscribe mail
-    await this.sendMailForSubscriptionEvent(SubscriptionEvent.SUBSCRIBE, subscription, {})
+      const user = await this.prisma.user.findUnique({
+        where: {
+          id: subscription.userID
+        },
+        select: unselectPassword
+      })
+      const invoice = await this.prisma.invoice.create({
```

Comment:
@tomaszdurka where do you create the periods? And are here not already existing functions for this? Code duplication?